bin/reth/src/debug_cmd/in_memory_merkle.rs

```diff
@@ -166,9 +166,9 @@ impl Command {
         let merkle_block_td =
             provider.header_td_by_number(merkle_block_number)?.unwrap_or_default();
         executor.execute_and_verify_receipt(
-            &block.clone().unseal(),
+            //todo
+            &block.clone().unseal().with_recovered_senders().unwrap(),
```

Comment:
Not sure what error to return here

Reply:
this should map to `BlockValidationError::SenderRecoveryError`